CHANGELOG.md

```diff
@@ -8,10 +8,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Added validation of the master presence in replicaset and the 
```

Comment:
I think it belongs to the `Fixed`

Reply:
Done. Sorry for the wait, there were some problems with rebase